### PR TITLE
Fix bug with mmcli interface

### PR DESCRIPTION
**Pull request by olivier-soracom**
@c9katayama I noticed a bug in the usage of mmcli interface and I have commited some code to fix it in the new branch. Can you consider for merging with master?

### CODE REVIEW

File: src/main/resources/soracom-endorse-version [hunk 1]
```diff
@@ -1 +1 @@
-Version: 0.1.0
\ No newline at end of file
+version = '00.03.02'
```
Comment (c9katayama) [inline src/main/resources/soracom-endorse-version]: Can we use "Version: 0.2.0" instead of version = '00.03.02'  ?

**Review by c9katayama — COMMENTED** [review]
I added a comment about versioning.

File: src/main/resources/soracom-endorse-version (hunk 1 above)
Comment (olivier-soracom) [reply src/main/resources/soracom-endorse-version]: Right. I changed the version number.

**Review by c9katayama — APPROVED** [review]

**Comment by c9katayama** [comment]
LGTM, thank you!

**Pull request merged**